### DISCUSSION TITLE
What are the options for migrating code between frameworks like hapi to express??

### DISCUSSION THREAD

**Discussion by vivekportpro** [community/community#131087] [Programming Help]
### Body

My project implemented in hapi.js and i want to migrate my code into express so problem is my project has many files and many features, i have to rewrite whole code again. so i am looking for any alternative solutions or tool??

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by mOnteySZEF** [comment]
Jol jol vivekportpro,
Migrating from **hapi.js** to **Express** will take some work. You can start by moving your routes since they are different in both frameworks writing small functions to bridge your **hapi** and **Express** code might help too. Also, check if there are any tools that can assist with the migration. **good luck**
